doc/intro-to-autoinstall.rst

```diff
@@ -11,15 +11,15 @@ This format is supported in the following installers:
  * Ubuntu Server, version 20.04 and later
  * Ubuntu Desktop, version 23.04 and later
 
-Autoinstallation lets you answer all those configuration questions ahead of
-time with an *autoinstall config* and lets the installation process run without
+Automatic installation lets you answer all those configuration questions ahead of
+time with an *autoinstall configuration* and lets the installation process run without
 any interaction.
 
 
-Differences from debian-installer preseeding
+Differences from :spellexception:`debian-installer` preseeding
 ============================================
 
-*preseeds* are the way to automate an installer based on debian-installer
+*preseeds* are the way to automate an installer based on :spellexception:`debian-installer`
```

Comment:
We might have more references to debian-installer in the future, this would be a good one to add to a wordlist.

Reply:
The spellcheck is dumb... It sees "debian", which I don't want to have in the allow-list. But you're right -- adding 'debian-installer'.